frontend/src/app/modules/result-selection/components/result-selection-job-group/result-selection-job-group.component.html

```diff
@@ -0,0 +1,25 @@
+<div class="card form-horizontal">
+  <h2>{{ 'frontend.de.iteratec.osm.resultSelection.jobGroup.title' | translate }}</h2>
+  <div class="form-group">
+    <select multiple class="form-control" id='job-groups-selection'>
```

Comment:
```suggestion
    <select multiple class="form-control" id='job-groups-selection' [(ngModel)]="selectedJobGroups">
```
By adding a ngModel control you can easily get the selected JobGroups in the component.